lib/processor/subprocess_test.go

```diff
@@ -136,7 +136,7 @@ func TestSubprocessLineBreaks(t *testing.T) {
 	})
 	msgs, res := proc.ProcessMessage(msgIn)
 	if len(msgs) != 1 {
-		t.Fatal("Wrong count of messages")
+		t.Fatalf("Wrong count of messages %d", len(msgs))
```

Comment:
This test was failing locally for some reason. Not sure why

Reply:
Yeah these are bad unit tests as they rely on installed binaries. If you rebase on master I've got them disabled.